prog.go

```diff
@@ -613,6 +613,26 @@ func (p *Program) ID() (ProgramID, error) {
 	return ProgramID(info.id), nil
 }
 
+// ProgramStats holds statistics of a program.
```

Comment:
Could you elaborate on the meaning of the values in the struct? e.g. is `RuntimeNS` the total accumulated runtime of the program? Or the latest invocation? Or the average?